superset/jinja_context.py

```diff
@@ -186,6 +188,28 @@ def url_param(
         return result
 
 
+def safe_proxy(func: Callable[..., Any], *args: Any, **kwargs: Any) -> Any:
+    none_type = type(None).__name__
+    allowed_types = [
```

Comment:
probably better to declare as `ALLOWED_TYPES` in module scope as it will never change